### PR TITLE
Make bucket executor default.

### PR DESCRIPTION
**Pull request by baldersheim**
@geirst or @vekterli PR

### CODE REVIEW

**Comment by baldersheim** [comment]
Build failure was unrelated

File: searchcore/src/vespa/searchcore/config/proton.def [hunk 1]
```diff
@@ -319,7 +319,7 @@ documentdb[].allocation.amortizecount int default=10000
 documentdb[].allocation.multivaluegrowfactor double default=0.2
 
 ## The ratio of used bytes that can be dead before attempting to perform compaction.
-documentdb[].allocation.max_dead_bytes_ratio double default=0.15
+documentdb[].allocation.max_dead_bytes_ratio double default=0.05
```
Comment (toregge) [inline searchcore/src/vespa/searchcore/config/proton.def:322]: Consider also updating search::CompactionStrategy() default constructor.

**Review by geirst — APPROVED** [review]
👍

File: searchcore/src/vespa/searchcore/config/proton.def (hunk 1 above)
Comment (baldersheim) [reply searchcore/src/vespa/searchcore/config/proton.def:322]: Done

**Pull request merged**